gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageTest.java

```diff
@@ -3457,3 +3544,32 @@ public int read() throws IOException {
     };
   }
 }
+
+
+class FaKeAccessTokenProviderWithCounter implements AccessTokenProvider {
```

Comment:
"Fake" instead of "FaKe"

Reply:
This class will be removed in the next iteration.